airflow/providers/amazon/aws/hooks/glue.py

```diff
@@ -107,8 +107,8 @@ def get_iam_execution_role(self) -> Dict:
             glue_execution_role = iam_client.get_role(RoleName=self.role_name)
             self.log.info("Iam Role Name: %s", self.role_name)
             return glue_execution_role
-        except Exception as general_error:
-            self.log.error("Failed to create aws glue job, error: %s", general_error)
+        except Exception:
+            self.log.error("Failed to create aws glue job.")
```

Comment:
Do we want to use `exception` here? (and those in the same file)

Reply:
Yep. Sounds strange. We shoud catch specific exception that we know about and let all the rest buble up directly, there is no point in extra logging here unless we want to provide any "specific" information resulting in helping the user to react to some known exceptions. There is no point in logging meaningless log here - the user knows, Glue job failed to be created already and providing this extra line with no actual "Help" for the user and instructions on what to do is borderline harrasing the user "Hello you already know we failed, so let us repeat it here").

Reply:
>Do we want to use exception here? (same for many below)

Generally if there was a `raise` I stuck to using `error` so the traceback wasn't logged twice.

Oof yeah @potiuk there are a _lot_ of generic exceptions in these files. I'll clean them up.